packages/backend-app-api/src/services/implementations/httpRouter/createHealthcheck.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Request, Response } from 'express';
+import Router from 'express-promise-router';
+
+export function createHealthcheck() {
+  const router = Router();
+  let handler = () => Promise.resolve({ status: 'ok' });
+
+  router.get('/healthcheck', async (_request: Request, response: Response) => {
```

Comment:
Thinking we should have this separated from the main routes, probably something like `/.backstage/health`, but potentially grouped a bit